mdbenchmark/analyze.py

```diff
@@ -142,20 +143,17 @@ def analyze(directory, plot, ncores):
         df.loc[i] = analyze_run(sim)
 
     if df.isnull().values.any():
-        click.echo(
-            '{}\tWe were not able to gather informations for all systems.\n\t'
+        console.warn(
+            '\tWe were not able to gather informations for all systems.\n\t'
```

Comment:
why start with a tab here?

Reply:
We are printing several lines to the user here. I wanted everything to be indented after the `WARNING` prefix, so we also need to indent the very first line with a tab.

Reply:
you can also just print one line. I don't see a reason why this sentence should be broken up on the terminal.